docs/app/styles/typography/typography-demo.component.html

```diff
@@ -2,7 +2,7 @@
     <h1>Typography Guidelines</h1>
     <h6>Last updated {{lastModified | date:'longDate'}}</h6>
 
-    <hc-tile>
+    <hc-tile class="width-overflow">
```

Comment:
It might be nice to have these responsive styles baked into the components themselves rather than requiring the developer to create a class like `width-overflow` to make them responsive.

Reply:
you could do something like the following
```scss
@include media-breakpoint-down(sm) {
     hc-tile {
     overflow-x: auto;
}}```